examples/kubernetes/connectivity-check/defaults.cue

```diff
@@ -0,0 +1,63 @@
+package connectivity_check
+
+// Default parameters for echo clients (may be overridden).
+deployment: [ID=_]: {
+	// General pod parameters
+	if ID =~ "^pod-to-[-_a-zA-Z0-9]*$" || ID =~ "^host-to-[-_a-zA-Z0-9]*$" {
+		_image: "docker.io/byrnedo/alpine-curl:0.1.8"
+		_command: ["/bin/ash", "-c", "sleep 1000000000"]
+	}
+
+	// readinessProbe target name
+	if ID =~ "^pod-to-a.*$" || ID =~ "^host-to-a.*$" {
+		_probeTarget: *"echo-a" | string
+	}
+	if ID =~ "^pod-to-b.*$" || ID =~ "^host-to-b.*$" {
+		_probeTarget: *"echo-b" | string
+	}
+	if ID =~ "^pod-to-c.*$" || ID =~ "^host-to-c.*$" {
+		_probeTarget: *"echo-c" | string
+	}
+}
+
+// Default parameters for echo clients (final).
+deployment: [ID=_]: {
+	// Topology
+	if ID =~ "^[-_a-zA-Z0-9]*intra-node[-_a-zA-Z0-9]*$" {
```

Comment:
question - any particular reason why you are limiting the character set? If it's for render-time validation I think that's fine, but maybe we should validate these yamls via some kind of local validator as a build step?

Reply:
Maybe it can just be `.*` instead, I don't have a specific motivation to restrict the characterset.

Is it possible to validate locally? I have been using the standard kubectl validation (which requires the resources such as CNP to already be configured in the connected cluster with schema validation included).